docs/ens.md

```diff
@@ -0,0 +1,107 @@
+---
+id: ens
+title: Adding accounts to an ENS domain
+sidebar_label: Using ENS with DOT/KSM accounts
+---
+
+ENS (Ethereum Name Service) is a system of smart contracts on the Ethereum blockchain which allows
+users to claim domain names like `bruno.eth`. Supporting wallets can then allow senders to input ENS
+domains instead of long and unwieldy addresses. This prevents phishing, fraud, typos, and adds a
+layer of usability on top of the regular wallet user experience.
+
+> Note: You will need an ENS name and an Ethereum account with some ether in it to follow along with
+> this guide. To register an ENS name, visit the [ENS App](https://app.ens.domains) or any number of
+> subdomain registrars like [Nameth](https://nameth.io). You will also need some way to use the
+> Ethereum address - following this guide on a personal computer is recommended. Wallets like
+> [Frame](https://frame.sh/) and [Metamask](https://metamask.io) are safe and will make interacting
+> with the Ethereum blockchain through your browser very easy.
+
+Despite living on the Ethereum blockchain, the ENS system has multi-chain support. In this guide
+you'll go through the process of adding a KSM and DOT address to ENS. We cover both KSM and DOT to
+show two different approaches.
+
+> Note: DOT can currently only be added using the Resolver method. KSM can be added through both
+> methods described below.
+
+This guide is also available in video format [on Youtube](https://youtu.be/XKjZk-5_mQc).
+
```

Comment:
I had to upgrade my Resolver before I could do anything.  Maybe make a mention of that here?